src/app/public/src/modules/shared/route-metadata.service.ts

```diff
@@ -18,12 +18,17 @@ export class StacheRouteMetadataService {
   constructor(
     @Inject(STACHE_ROUTE_METADATA_SERVICE_CONFIG)
     public metadata: any[]) {
-      this.metadata.forEach(route => {
-        if (route.order) {
-          route.order = numberConverter(route.order);
-        }
-      });
+      this.metadata.forEach((route: any) => this.validateNavOrder(route));
     }
+
+  private validateNavOrder(route: any): void {
+    if (route.hasOwnProperty('order')) {
+      route.order = numberConverter(route.order);
+      if (route.order <= 0 || route.order !== parseInt(route.order, 10)) {
```

Comment:
Yeah, I'm in favor of not supporting 0 in this case too. It makes more sense for what it's doing on the page, and zero based indexes aren't as intuitive to non-technical users.